{
 "cells": [
  {
   "cell_type": "markdown",
   "id": "a7adc07c-0e94-4ccb-94f6-73bf8c303984",
   "metadata": {},
   "source": [
    "# Getting statistics on the CKG graph\n",
    "\n",
    "The only difference between this and the SPOKE notebook is that I'm currently running the CKG neo4j."
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "32c96888-ef56-4935-bb21-9117c39d631b",
   "metadata": {},
   "outputs": [],
   "source": [
    "from neo4j import GraphDatabase\n",
    "import pandas as pd"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "60fc9ec1-94a6-4f0e-a9b3-21eda264eda5",
   "metadata": {},
   "outputs": [],
   "source": [
    "driver = GraphDatabase.driver(uri='bolt://localhost:7687')"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "02ed8871-7e95-43cd-a574-0c3fb224974d",
   "metadata": {},
   "outputs": [],
   "source": [
    "# transactions\n",
    "def get_node_sources(tx):\n",
    "    result = tx.run('MATCH (n) RETURN LABELS(n), n.source, COUNT(ID(n));')\n",
    "    return list(result)\n",
    "\n",
    "def get_node_types(tx):\n",
    "    return list(tx.run('MATCH (n) RETURN LABELS(n), COUNT(ID(n));'))\n",
    "\n",
    "def get_edge_types(tx):\n",
    "    return list(tx.run('MATCH ()-[e]-() RETURN TYPE(e), COUNT(e);'))\n",
    "\n",
    "def get_edge_sources(tx):\n",
    "    return list(tx.run('MATCH ()-[e]-() RETURN TYPE(e), e.source, e.sources[0], COUNT(e);'))"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "1abfbe1e-0fe4-48c9-8806-be06fc10d32b",
   "metadata": {},
   "outputs": [],
   "source": [
    "with driver.session() as session:\n",
    "    node_sources = session.execute_read(get_node_sources)\n",
    "    edge_types = session.execute_read(get_edge_types)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 9,
   "id": "b38d7457-43fe-4ad1-b022-44cb25258792",
   "metadata": {},
   "outputs": [],
   "source": [
    "with driver.session() as session:\n",
    "    edge_sources = session.execute_read(get_edge_sources)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "5a396f38-837f-4fdf-9ff1-79cb339bd52a",
   "metadata": {},
   "outputs": [],
   "source": [
    "node_sources_list = [x.data() for x in node_sources]\n",
    "node_sources_list = [(x['LABELS(n)'][0], x['n.source'], x['COUNT(ID(n))']) for x in node_sources_list]\n",
    "node_sources_table = pd.DataFrame(node_sources_list, columns=['label', 'source', 'count'])\n",
    "node_sources_table = node_sources_table.sort_values('label')\n",
    "node_sources_table = node_sources_table.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "423abaae-e816-4e89-8dea-6fca530ec528",
   "metadata": {},
   "source": [
    "## Node statistics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "912f2194-e176-4c5d-a699-b1a98c70a87e",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>label</th>\n",
       "      <th>source</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>Amino_acid_sequence</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>20614</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>Analytical_sample</td>\n",
       "      <td>None</td>\n",
       "      <td>172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>Biological_process</td>\n",
       "      <td>None</td>\n",
       "      <td>28642</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>Biological_sample</td>\n",
       "      <td>None</td>\n",
       "      <td>170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>Cellular_component</td>\n",
       "      <td>None</td>\n",
       "      <td>4176</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>Chromosome</td>\n",
       "      <td>None</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>Clinically_relevant_variant</td>\n",
       "      <td>CGI</td>\n",
       "      <td>6856</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>Clinically_relevant_variant</td>\n",
       "      <td>None</td>\n",
       "      <td>183478</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>Complex</td>\n",
       "      <td>CORUM</td>\n",
       "      <td>2700</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>Disease</td>\n",
       "      <td>None</td>\n",
       "      <td>10791</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>Experiment</td>\n",
       "      <td>None</td>\n",
       "      <td>2829</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>Experimental_factor</td>\n",
       "      <td>None</td>\n",
       "      <td>9883</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>Food</td>\n",
       "      <td>FooDB</td>\n",
       "      <td>992</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>Functional_region</td>\n",
       "      <td>PFam</td>\n",
       "      <td>16169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>GWAS_study</td>\n",
       "      <td>None</td>\n",
       "      <td>8713</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>Gene</td>\n",
       "      <td>None</td>\n",
       "      <td>42571</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>Known_variant</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>10630108</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>Metabolite</td>\n",
       "      <td>None</td>\n",
       "      <td>114222</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>Modification</td>\n",
       "      <td>None</td>\n",
       "      <td>1978</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>Modified_protein</td>\n",
       "      <td>SIGNOR</td>\n",
       "      <td>621</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>Modified_protein</td>\n",
       "      <td>experimentally_identified</td>\n",
       "      <td>20786</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>Molecular_function</td>\n",
       "      <td>None</td>\n",
       "      <td>11169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>Pathway</td>\n",
       "      <td>SMPDB</td>\n",
       "      <td>48703</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>Pathway</td>\n",
       "      <td>Reactome</td>\n",
       "      <td>2516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>Peptide</td>\n",
       "      <td>None</td>\n",
       "      <td>1001105</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>Phenotype</td>\n",
       "      <td>None</td>\n",
       "      <td>15872</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>Project</td>\n",
       "      <td>None</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>Protein</td>\n",
       "      <td>None</td>\n",
       "      <td>228725</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>Protein_structure</td>\n",
       "      <td>Uniprot</td>\n",
       "      <td>49317</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>Publication</td>\n",
       "      <td>None</td>\n",
       "      <td>1791712</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>Subject</td>\n",
       "      <td>None</td>\n",
       "      <td>169</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>Tissue</td>\n",
       "      <td>None</td>\n",
       "      <td>5897</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>Transcript</td>\n",
       "      <td>None</td>\n",
       "      <td>280910</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>Units</td>\n",
       "      <td>None</td>\n",
       "      <td>442</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>User</td>\n",
       "      <td>None</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                          label                     source     count\n",
       "0           Amino_acid_sequence                    UniProt     20614\n",
       "1             Analytical_sample                       None       172\n",
       "2            Biological_process                       None     28642\n",
       "3             Biological_sample                       None       170\n",
       "4            Cellular_component                       None      4176\n",
       "5                    Chromosome                       None        25\n",
       "6   Clinically_relevant_variant                        CGI      6856\n",
       "7   Clinically_relevant_variant                       None    183478\n",
       "8                       Complex                      CORUM      2700\n",
       "9                       Disease                       None     10791\n",
       "10                   Experiment                       None      2829\n",
       "11          Experimental_factor                       None      9883\n",
       "12                         Food                      FooDB       992\n",
       "13            Functional_region                       PFam     16169\n",
       "14                   GWAS_study                       None      8713\n",
       "15                         Gene                       None     42571\n",
       "16                Known_variant                    UniProt  10630108\n",
       "17                   Metabolite                       None    114222\n",
       "18                 Modification                       None      1978\n",
       "19             Modified_protein                     SIGNOR       621\n",
       "20             Modified_protein  experimentally_identified     20786\n",
       "21           Molecular_function                       None     11169\n",
       "22                      Pathway                      SMPDB     48703\n",
       "23                      Pathway                   Reactome      2516\n",
       "24                      Peptide                       None   1001105\n",
       "25                    Phenotype                       None     15872\n",
       "26                      Project                       None         7\n",
       "27                      Protein                       None    228725\n",
       "28            Protein_structure                    Uniprot     49317\n",
       "29                  Publication                       None   1791712\n",
       "30                      Subject                       None       169\n",
       "31                       Tissue                       None      5897\n",
       "32                   Transcript                       None    280910\n",
       "33                        Units                       None       442\n",
       "34                         User                       None         2"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "node_sources_table"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "0b18b3a5-5343-4250-a2e4-e53137f42c3f",
   "metadata": {},
   "source": [
    "## Edge statistics"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "id": "1b6919eb-c25e-4a83-a87e-877f9ef32b7c",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>edge_type</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ACTS_ON</td>\n",
       "      <td>1977410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ANNOTATED_IN_PATHWAY</td>\n",
       "      <td>2407618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>33415258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>BELONGS_TO_PROTEIN</td>\n",
       "      <td>7258116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>BELONGS_TO_SUBJECT</td>\n",
       "      <td>340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>COMPILED_INTERACTS_WITH</td>\n",
       "      <td>3913224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>CURATED_AFFECTS_INTERACTION_WITH</td>\n",
       "      <td>21746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>595961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>DETECTED_IN_PATHOLOGY_SAMPLE</td>\n",
       "      <td>3394496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>FOUND_IN_PROTEIN</td>\n",
       "      <td>408488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>HAS_ENROLLED</td>\n",
       "      <td>338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>HAS_MODIFICATION</td>\n",
       "      <td>42814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>HAS_MODIFIED_SITE</td>\n",
       "      <td>42842</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>HAS_PARENT</td>\n",
       "      <td>256698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>HAS_QUANTIFIED_MODIFIED_PROTEIN</td>\n",
       "      <td>448956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>HAS_QUANTIFIED_PROTEIN</td>\n",
       "      <td>1595302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>HAS_SEQUENCE</td>\n",
       "      <td>41228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>HAS_STRUCTURE</td>\n",
       "      <td>391280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>IS_BIOMARKER_OF_DISEASE</td>\n",
       "      <td>1030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>IS_QCMARKER_IN_TISSUE</td>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>IS_RESPONSIBLE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>IS_SUBSTRATE_OF</td>\n",
       "      <td>1988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>IS_SUBUNIT_OF</td>\n",
       "      <td>21936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>LOCATED_IN</td>\n",
       "      <td>591824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>MAPS_TO</td>\n",
       "      <td>4578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>MENTIONED_IN_PUBLICATION</td>\n",
       "      <td>222218476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>PARTICIPATES_IN</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>PUBLISHED_IN</td>\n",
       "      <td>7878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>SPLITTED_INTO</td>\n",
       "      <td>344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>STUDIES_DISEASE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>STUDIES_TISSUE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>STUDIES_TRAIT</td>\n",
       "      <td>18500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>TRANSCRIBED_INTO</td>\n",
       "      <td>516974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>TRANSLATED_INTO</td>\n",
       "      <td>748588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>VARIANT_FOUND_IN_CHROMOSOME</td>\n",
       "      <td>21260216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>VARIANT_FOUND_IN_GENE</td>\n",
       "      <td>21277870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>VARIANT_FOUND_IN_GWAS</td>\n",
       "      <td>32256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>VARIANT_FOUND_IN_PROTEIN</td>\n",
       "      <td>53614586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>VARIANT_IS_CLINICALLY_RELEVANT</td>\n",
       "      <td>338</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           edge_type      count\n",
       "0                            ACTS_ON    1977410\n",
       "1               ANNOTATED_IN_PATHWAY    2407618\n",
       "2                    ASSOCIATED_WITH   33415258\n",
       "3                 BELONGS_TO_PROTEIN    7258116\n",
       "4                 BELONGS_TO_SUBJECT        340\n",
       "5            COMPILED_INTERACTS_WITH    3913224\n",
       "6   CURATED_AFFECTS_INTERACTION_WITH      21746\n",
       "7             CURATED_INTERACTS_WITH     595961\n",
       "8       DETECTED_IN_PATHOLOGY_SAMPLE    3394496\n",
       "9                   FOUND_IN_PROTEIN     408488\n",
       "10                      HAS_ENROLLED        338\n",
       "11                  HAS_MODIFICATION      42814\n",
       "12                 HAS_MODIFIED_SITE      42842\n",
       "13                        HAS_PARENT     256698\n",
       "14   HAS_QUANTIFIED_MODIFIED_PROTEIN     448956\n",
       "15            HAS_QUANTIFIED_PROTEIN    1595302\n",
       "16                      HAS_SEQUENCE      41228\n",
       "17                     HAS_STRUCTURE     391280\n",
       "18           IS_BIOMARKER_OF_DISEASE       1030\n",
       "19             IS_QCMARKER_IN_TISSUE        498\n",
       "20                    IS_RESPONSIBLE         14\n",
       "21                   IS_SUBSTRATE_OF       1988\n",
       "22                     IS_SUBUNIT_OF      21936\n",
       "23                        LOCATED_IN     591824\n",
       "24                           MAPS_TO       4578\n",
       "25          MENTIONED_IN_PUBLICATION  222218476\n",
       "26                   PARTICIPATES_IN         14\n",
       "27                      PUBLISHED_IN       7878\n",
       "28                     SPLITTED_INTO        344\n",
       "29                   STUDIES_DISEASE         14\n",
       "30                    STUDIES_TISSUE         14\n",
       "31                     STUDIES_TRAIT      18500\n",
       "32                  TRANSCRIBED_INTO     516974\n",
       "33                   TRANSLATED_INTO     748588\n",
       "34       VARIANT_FOUND_IN_CHROMOSOME   21260216\n",
       "35             VARIANT_FOUND_IN_GENE   21277870\n",
       "36             VARIANT_FOUND_IN_GWAS      32256\n",
       "37          VARIANT_FOUND_IN_PROTEIN   53614586\n",
       "38    VARIANT_IS_CLINICALLY_RELEVANT        338"
      ]
     },
     "execution_count": 7,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "edge_types_list = [x.data() for x in edge_types]\n",
    "edge_types_list = [(x['TYPE(e)'], x['COUNT(e)']) for x in edge_types_list]\n",
    "edge_types_table = pd.DataFrame(edge_types_list, columns=['edge_type', 'count'])\n",
    "edge_types_table = edge_types_table.sort_values('edge_type')\n",
    "edge_types_table = edge_types_table.reset_index(drop=True)\n",
    "edge_types_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "8bb8906f-d00c-43f3-aee2-8fde53b33d32",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>edge_type</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>MENTIONED_IN_PUBLICATION</td>\n",
       "      <td>222218476</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>VARIANT_FOUND_IN_PROTEIN</td>\n",
       "      <td>53614586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>33415258</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>VARIANT_FOUND_IN_GENE</td>\n",
       "      <td>21277870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>VARIANT_FOUND_IN_CHROMOSOME</td>\n",
       "      <td>21260216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>BELONGS_TO_PROTEIN</td>\n",
       "      <td>7258116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>COMPILED_INTERACTS_WITH</td>\n",
       "      <td>3913224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>DETECTED_IN_PATHOLOGY_SAMPLE</td>\n",
       "      <td>3394496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ANNOTATED_IN_PATHWAY</td>\n",
       "      <td>2407618</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>ACTS_ON</td>\n",
       "      <td>1977410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>HAS_QUANTIFIED_PROTEIN</td>\n",
       "      <td>1595302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>TRANSLATED_INTO</td>\n",
       "      <td>748588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>595961</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>LOCATED_IN</td>\n",
       "      <td>591824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>TRANSCRIBED_INTO</td>\n",
       "      <td>516974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>HAS_QUANTIFIED_MODIFIED_PROTEIN</td>\n",
       "      <td>448956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>FOUND_IN_PROTEIN</td>\n",
       "      <td>408488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>HAS_STRUCTURE</td>\n",
       "      <td>391280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>HAS_PARENT</td>\n",
       "      <td>256698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>HAS_MODIFIED_SITE</td>\n",
       "      <td>42842</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>HAS_MODIFICATION</td>\n",
       "      <td>42814</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>HAS_SEQUENCE</td>\n",
       "      <td>41228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>VARIANT_FOUND_IN_GWAS</td>\n",
       "      <td>32256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>IS_SUBUNIT_OF</td>\n",
       "      <td>21936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>CURATED_AFFECTS_INTERACTION_WITH</td>\n",
       "      <td>21746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>STUDIES_TRAIT</td>\n",
       "      <td>18500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>PUBLISHED_IN</td>\n",
       "      <td>7878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>MAPS_TO</td>\n",
       "      <td>4578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>IS_SUBSTRATE_OF</td>\n",
       "      <td>1988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>IS_BIOMARKER_OF_DISEASE</td>\n",
       "      <td>1030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>IS_QCMARKER_IN_TISSUE</td>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>SPLITTED_INTO</td>\n",
       "      <td>344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>BELONGS_TO_SUBJECT</td>\n",
       "      <td>340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>HAS_ENROLLED</td>\n",
       "      <td>338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>VARIANT_IS_CLINICALLY_RELEVANT</td>\n",
       "      <td>338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>STUDIES_TISSUE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>STUDIES_DISEASE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>PARTICIPATES_IN</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>IS_RESPONSIBLE</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                           edge_type      count\n",
       "0           MENTIONED_IN_PUBLICATION  222218476\n",
       "1           VARIANT_FOUND_IN_PROTEIN   53614586\n",
       "2                    ASSOCIATED_WITH   33415258\n",
       "3              VARIANT_FOUND_IN_GENE   21277870\n",
       "4        VARIANT_FOUND_IN_CHROMOSOME   21260216\n",
       "5                 BELONGS_TO_PROTEIN    7258116\n",
       "6            COMPILED_INTERACTS_WITH    3913224\n",
       "7       DETECTED_IN_PATHOLOGY_SAMPLE    3394496\n",
       "8               ANNOTATED_IN_PATHWAY    2407618\n",
       "9                            ACTS_ON    1977410\n",
       "10            HAS_QUANTIFIED_PROTEIN    1595302\n",
       "11                   TRANSLATED_INTO     748588\n",
       "12            CURATED_INTERACTS_WITH     595961\n",
       "13                        LOCATED_IN     591824\n",
       "14                  TRANSCRIBED_INTO     516974\n",
       "15   HAS_QUANTIFIED_MODIFIED_PROTEIN     448956\n",
       "16                  FOUND_IN_PROTEIN     408488\n",
       "17                     HAS_STRUCTURE     391280\n",
       "18                        HAS_PARENT     256698\n",
       "19                 HAS_MODIFIED_SITE      42842\n",
       "20                  HAS_MODIFICATION      42814\n",
       "21                      HAS_SEQUENCE      41228\n",
       "22             VARIANT_FOUND_IN_GWAS      32256\n",
       "23                     IS_SUBUNIT_OF      21936\n",
       "24  CURATED_AFFECTS_INTERACTION_WITH      21746\n",
       "25                     STUDIES_TRAIT      18500\n",
       "26                      PUBLISHED_IN       7878\n",
       "27                           MAPS_TO       4578\n",
       "28                   IS_SUBSTRATE_OF       1988\n",
       "29           IS_BIOMARKER_OF_DISEASE       1030\n",
       "30             IS_QCMARKER_IN_TISSUE        498\n",
       "31                     SPLITTED_INTO        344\n",
       "32                BELONGS_TO_SUBJECT        340\n",
       "33                      HAS_ENROLLED        338\n",
       "34    VARIANT_IS_CLINICALLY_RELEVANT        338\n",
       "35                    STUDIES_TISSUE         14\n",
       "36                   STUDIES_DISEASE         14\n",
       "37                   PARTICIPATES_IN         14\n",
       "38                    IS_RESPONSIBLE         14"
      ]
     },
     "execution_count": 8,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "edge_types_table = edge_types_table.sort_values('count', ascending=False)\n",
    "edge_types_table = edge_types_table.reset_index(drop=True)\n",
    "edge_types_table"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "9a7c9795-738e-40bc-80f9-f8bab1de8a34",
   "metadata": {},
   "source": [
    "## Edge sources"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 10,
   "id": "e2534b5d-dd12-4c60-95b4-efd1ea18c101",
   "metadata": {},
   "outputs": [],
   "source": [
    "edge_sources_list = [x.data() for x in edge_sources]\n",
    "edge_sources_list = [(x['TYPE(e)'], x['e.source'], x['e.sources[0]'], x['COUNT(e)']) for x in edge_sources_list]\n",
    "edge_sources_table = pd.DataFrame(edge_sources_list, columns=['edge_type', 'source', 'sources', 'count'])\n",
    "edge_sources_table = edge_sources_table.sort_values('edge_type')\n",
    "edge_sources_table = edge_sources_table.reset_index(drop=True)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 11,
   "id": "be6621a7-1701-4a83-8403-c0d74818a1f9",
   "metadata": {},
   "outputs": [],
   "source": [
    "edge_sources_table['source'] = edge_sources_table['source'].fillna(edge_sources_table['sources'])\n",
    "del edge_sources_table['sources']"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 12,
   "id": "3e145ce3-4074-415b-b883-da824749d2d6",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>edge_type</th>\n",
       "      <th>source</th>\n",
       "      <th>count</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>ACTS_ON</td>\n",
       "      <td>STRING</td>\n",
       "      <td>1977410</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>ANNOTATED_IN_PATHWAY</td>\n",
       "      <td>Reactome</td>\n",
       "      <td>117270</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>ANNOTATED_IN_PATHWAY</td>\n",
       "      <td>SMPDB</td>\n",
       "      <td>2290348</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: ORPHANET;UNIPROT</td>\n",
       "      <td>52</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;GENOMICS_ENGLAND;ORPHANET;UN...</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;UNIPROT</td>\n",
       "      <td>106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;GENOMICS_ENGLAND</td>\n",
       "      <td>128</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLAND</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;GENOMICS_ENGLAND;UNIPROT</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;GENOMICS_ENGLAND;UNIPROT</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;ORPHANET</td>\n",
       "      <td>34</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: PSYGENET</td>\n",
       "      <td>9634</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>12</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;UNIPROT</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>13</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;U...</td>\n",
       "      <td>226</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>14</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;O...</td>\n",
       "      <td>298</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>15</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;ORPHANET</td>\n",
       "      <td>66</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>16</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;ORPHANET;UNIPROT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>17</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;ORPHA...</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>18</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: GENOMICS_ENGLAND;ORPHANET;UNIPROT</td>\n",
       "      <td>178</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>19</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;ORPHANET</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>20</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CLINGEN;CTD_human</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>21</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CLINGEN;CTD_human;ORPHANET</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>22</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;UNIPROT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>23</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLA...</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>24</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: GENOMICS_ENGLAND;PSYGENET</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>25</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;ORPHANET;UNIPROT</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>26</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLA...</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>27</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>TISSUES</td>\n",
       "      <td>13058452</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>28</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>672516</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>29</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>CORUM</td>\n",
       "      <td>9540</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>30</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>COMPARTMENTS</td>\n",
       "      <td>7124610</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>31</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>OncoKB</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>32</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human</td>\n",
       "      <td>296</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>33</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;PSYGENET</td>\n",
       "      <td>566</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>34</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;ORPHANET</td>\n",
       "      <td>90</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>35</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DISEASES</td>\n",
       "      <td>8931224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>36</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>CGI</td>\n",
       "      <td>8924</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>37</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: BEFREE</td>\n",
       "      <td>1688338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>38</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human</td>\n",
       "      <td>87860</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>39</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human</td>\n",
       "      <td>802</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>40</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: GENOMICS_ENGLAND</td>\n",
       "      <td>14804</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>41</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: ORPHANET</td>\n",
       "      <td>7964</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;ORPHANET</td>\n",
       "      <td>1590</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>43</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;GENOMICS_ENGLAND;UNIPROT</td>\n",
       "      <td>2598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>44</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;GENOMICS_ENGLAND;ORPHANET</td>\n",
       "      <td>248</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>45</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: GENOMICS_ENGLAND;ORPHANET</td>\n",
       "      <td>1306</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>46</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;GENOMICS_ENGLAND;ORPHANET;...</td>\n",
       "      <td>2530</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>47</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;GENOMICS_ENGLAND</td>\n",
       "      <td>3090</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>48</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN</td>\n",
       "      <td>3392</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>49</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: UNIPROT</td>\n",
       "      <td>5028</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>50</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;GENOMICS_ENGLAND;ORPHANET</td>\n",
       "      <td>1548</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>51</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;ORPHANET</td>\n",
       "      <td>358</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>52</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;GENOMICS_ENGLAND</td>\n",
       "      <td>186</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>53</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI</td>\n",
       "      <td>2506</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>54</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND</td>\n",
       "      <td>54</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>55</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;UNIPROT</td>\n",
       "      <td>662</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>56</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;UNIPROT</td>\n",
       "      <td>126</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>57</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;O...</td>\n",
       "      <td>260</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>58</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND</td>\n",
       "      <td>216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>59</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>None</td>\n",
       "      <td>1771568</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>60</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CTD_human;ORPHANET;UNIPROT</td>\n",
       "      <td>152</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>61</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: GENOMICS_ENGLAND;UNIPROT</td>\n",
       "      <td>784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62</th>\n",
       "      <td>ASSOCIATED_WITH</td>\n",
       "      <td>DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;UNIPROT</td>\n",
       "      <td>62</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>63</th>\n",
       "      <td>BELONGS_TO_PROTEIN</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>7258116</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>64</th>\n",
       "      <td>BELONGS_TO_SUBJECT</td>\n",
       "      <td>None</td>\n",
       "      <td>340</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>65</th>\n",
       "      <td>COMPILED_INTERACTS_WITH</td>\n",
       "      <td>[STRING]</td>\n",
       "      <td>3913224</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>66</th>\n",
       "      <td>CURATED_AFFECTS_INTERACTION_WITH</td>\n",
       "      <td>Intact-MutationDs</td>\n",
       "      <td>21746</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>67</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP, InnateDB, MINT]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>68</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, DIP, MINT]</td>\n",
       "      <td>37</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>69</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, matrixdb]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>70</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, matrixdb, DIP]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, I2D, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP, InnateDB, MINT]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>73</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, mbinfo]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>74</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, DIP, MINT]</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>75</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, IntAct, Molecular Connections, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>76</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, matrixdb, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>77</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, InnateDB]</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>78</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, DIP]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>79</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, I2D]</td>\n",
       "      <td>30</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, I2D, IntAct, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>81</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, IntAct, DIP, MINT]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>82</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, HPIDb]</td>\n",
       "      <td>82</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, HPIDb]</td>\n",
       "      <td>74</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>84</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, IntAct, I2D, MINT]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>85</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb, UniProt, IntAct, MINT]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb, IntAct, MINT]</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>87</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, HPIDb]</td>\n",
       "      <td>25</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>88</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, DIP, HPIDb]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>89</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb, UniProt, MINT]</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>90</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, DIP, MINT]</td>\n",
       "      <td>10</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>91</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct]</td>\n",
       "      <td>58</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>92</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, IntAct, DIP]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>93</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb, MINT]</td>\n",
       "      <td>18</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP, HPIDb]</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>95</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, I2D, DIP, MINT]</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>96</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, I2D, DIP, MINT, bhf-ucl]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>97</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, MINT, HPIDb]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, Molecular Connections, IntAct, mbinfo]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>99</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, DIP]</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, HPIDb]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>101</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mpidb]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP, HPIDb]</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>103</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, matrixdb, DIP]</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>104</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, bhf-ucl]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>105</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, HPIDb]</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>106</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, mbinfo]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP, matrixdb]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, IntAct, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>110</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, I2D, IntAct]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>111</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, DIP, I2D]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>112</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, I2D, MINT]</td>\n",
       "      <td>24</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>113</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, bhf-ucl, IntAct, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>114</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, I2D, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>115</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, InnateDB]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>116</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, InnateDB, MINT]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>117</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, Molecular Connections, I2D]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[NTNU, IntAct, I2D, MINT, bhf-ucl]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>119</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, I2D, DIP]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, UniProt, DIP, MINT, bhf-ucl]</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>121</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP, I2D]</td>\n",
       "      <td>5</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>122</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, Molecular Connections, IntAct]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>123</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[matrixdb, DIP]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>124</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, DIP]</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>125</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, DIP, HPIDb]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>126</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, IntAct, MINT]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>127</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, mpidb]</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>128</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, InnateDB]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>129</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[matrixdb]</td>\n",
       "      <td>293</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>130</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP]</td>\n",
       "      <td>1534</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>131</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb, IntAct, DIP, MINT]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>132</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, MINT]</td>\n",
       "      <td>12</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>133</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, Molecular Connections, IntAct, DIP]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl]</td>\n",
       "      <td>3332</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, InnateDB, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>136</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, MINT]</td>\n",
       "      <td>5360</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>137</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>138</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP]</td>\n",
       "      <td>9419</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>139</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D]</td>\n",
       "      <td>734</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>140</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[MINT]</td>\n",
       "      <td>55665</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>141</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, UniProt, DIP, InnateDB, MINT]</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>142</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt]</td>\n",
       "      <td>19157</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>143</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct]</td>\n",
       "      <td>494635</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>144</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, DIP]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>145</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, MINT, DIP, I2D]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>146</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, HPIDb]</td>\n",
       "      <td>6</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>147</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[MINT, HPIDb]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>148</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, I2D, UniProt, DIP, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>149</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[InnateDB, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>150</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, InnateDB, I2D]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>151</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[MINT, IntAct, UniProt, DIP, HPIDb]</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>152</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[MINT, IntAct, UniProt, InnateDB, HPIDb]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>153</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[matrixdb, MINT]</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>154</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, Molecular Connections]</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>155</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, IntAct, DIP]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>156</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, HPIDb, MINT]</td>\n",
       "      <td>1</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>157</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, IntAct]</td>\n",
       "      <td>86</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>158</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, IntAct, DIP]</td>\n",
       "      <td>16</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>159</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP, MINT]</td>\n",
       "      <td>446</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>160</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP, MINT]</td>\n",
       "      <td>361</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>161</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[NTNU]</td>\n",
       "      <td>46</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>162</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, DIP]</td>\n",
       "      <td>20</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>163</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, bhf-ucl]</td>\n",
       "      <td>9</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>164</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, IntAct, MINT]</td>\n",
       "      <td>26</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>165</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, IntAct, MINT]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>166</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[IntAct, DIP, InnateDB]</td>\n",
       "      <td>8</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>167</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, IntAct, MINT]</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>168</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, bhf-ucl, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>169</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, DIP, MINT]</td>\n",
       "      <td>133</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>170</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[NTNU, IntAct, MINT]</td>\n",
       "      <td>4</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>171</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, IntAct]</td>\n",
       "      <td>40</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>172</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct]</td>\n",
       "      <td>1413</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>173</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[InnateDB]</td>\n",
       "      <td>291</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>174</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, DIP]</td>\n",
       "      <td>151</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>175</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, I2D]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>176</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[I2D, MINT]</td>\n",
       "      <td>38</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>177</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, DIP]</td>\n",
       "      <td>170</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>178</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, MINT]</td>\n",
       "      <td>473</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>179</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, IntAct]</td>\n",
       "      <td>17</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>180</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[bhf-ucl, MINT]</td>\n",
       "      <td>32</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>181</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo]</td>\n",
       "      <td>209</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>182</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[mbinfo, IntAct, bhf-ucl, MINT]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[DIP, InnateDB]</td>\n",
       "      <td>7</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>184</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections]</td>\n",
       "      <td>336</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>185</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, DIP, I2D]</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>186</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[Molecular Connections, MINT]</td>\n",
       "      <td>22</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>187</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[UniProt, IntAct, MINT]</td>\n",
       "      <td>350</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>188</th>\n",
       "      <td>CURATED_INTERACTS_WITH</td>\n",
       "      <td>[HPIDb]</td>\n",
       "      <td>427</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>189</th>\n",
       "      <td>DETECTED_IN_PATHOLOGY_SAMPLE</td>\n",
       "      <td>Human Protein Atlas pathology</td>\n",
       "      <td>3394496</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>190</th>\n",
       "      <td>FOUND_IN_PROTEIN</td>\n",
       "      <td>PFam</td>\n",
       "      <td>408488</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>191</th>\n",
       "      <td>HAS_ENROLLED</td>\n",
       "      <td>None</td>\n",
       "      <td>338</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>192</th>\n",
       "      <td>HAS_MODIFICATION</td>\n",
       "      <td>None</td>\n",
       "      <td>41572</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>193</th>\n",
       "      <td>HAS_MODIFICATION</td>\n",
       "      <td>SIGNOR</td>\n",
       "      <td>1242</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>194</th>\n",
       "      <td>HAS_MODIFIED_SITE</td>\n",
       "      <td>None</td>\n",
       "      <td>42842</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>195</th>\n",
       "      <td>HAS_PARENT</td>\n",
       "      <td>None</td>\n",
       "      <td>256698</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>196</th>\n",
       "      <td>HAS_QUANTIFIED_MODIFIED_PROTEIN</td>\n",
       "      <td>None</td>\n",
       "      <td>448956</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>197</th>\n",
       "      <td>HAS_QUANTIFIED_PROTEIN</td>\n",
       "      <td>None</td>\n",
       "      <td>1595302</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>198</th>\n",
       "      <td>HAS_SEQUENCE</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>41228</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>199</th>\n",
       "      <td>HAS_STRUCTURE</td>\n",
       "      <td>UniProt</td>\n",
       "      <td>391280</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200</th>\n",
       "      <td>IS_BIOMARKER_OF_DISEASE</td>\n",
       "      <td>Nicolai Jacob Wewer Albrechtsen</td>\n",
       "      <td>1030</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>201</th>\n",
       "      <td>IS_QCMARKER_IN_TISSUE</td>\n",
       "      <td>None</td>\n",
       "      <td>498</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>202</th>\n",
       "      <td>IS_RESPONSIBLE</td>\n",
       "      <td>None</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>203</th>\n",
       "      <td>IS_SUBSTRATE_OF</td>\n",
       "      <td>SIGNOR</td>\n",
       "      <td>1988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>204</th>\n",
       "      <td>IS_SUBUNIT_OF</td>\n",
       "      <td>CORUM</td>\n",
       "      <td>21936</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>205</th>\n",
       "      <td>LOCATED_IN</td>\n",
       "      <td>None</td>\n",
       "      <td>591824</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>206</th>\n",
       "      <td>MAPS_TO</td>\n",
       "      <td>None</td>\n",
       "      <td>4578</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>207</th>\n",
       "      <td>MENTIONED_IN_PUBLICATION</td>\n",
       "      <td>PFam</td>\n",
       "      <td>4870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>208</th>\n",
       "      <td>MENTIONED_IN_PUBLICATION</td>\n",
       "      <td>None</td>\n",
       "      <td>222213606</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>209</th>\n",
       "      <td>PARTICIPATES_IN</td>\n",
       "      <td>None</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>210</th>\n",
       "      <td>PUBLISHED_IN</td>\n",
       "      <td>None</td>\n",
       "      <td>7878</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>211</th>\n",
       "      <td>SPLITTED_INTO</td>\n",
       "      <td>None</td>\n",
       "      <td>344</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>212</th>\n",
       "      <td>STUDIES_DISEASE</td>\n",
       "      <td>None</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>213</th>\n",
       "      <td>STUDIES_TISSUE</td>\n",
       "      <td>None</td>\n",
       "      <td>14</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>214</th>\n",
       "      <td>STUDIES_TRAIT</td>\n",
       "      <td>GWAS Catalog</td>\n",
       "      <td>18500</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>215</th>\n",
       "      <td>TRANSCRIBED_INTO</td>\n",
       "      <td>None</td>\n",
       "      <td>516974</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>216</th>\n",
       "      <td>TRANSLATED_INTO</td>\n",
       "      <td>None</td>\n",
       "      <td>748588</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>217</th>\n",
       "      <td>VARIANT_FOUND_IN_CHROMOSOME</td>\n",
       "      <td>None</td>\n",
       "      <td>21260216</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>218</th>\n",
       "      <td>VARIANT_FOUND_IN_GENE</td>\n",
       "      <td>None</td>\n",
       "      <td>21277870</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>219</th>\n",
       "      <td>VARIANT_FOUND_IN_GWAS</td>\n",
       "      <td>GWAS Catalog</td>\n",
       "      <td>32256</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>220</th>\n",
       "      <td>VARIANT_FOUND_IN_PROTEIN</td>\n",
       "      <td>None</td>\n",
       "      <td>53614586</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>221</th>\n",
       "      <td>VARIANT_IS_CLINICALLY_RELEVANT</td>\n",
       "      <td>CGI</td>\n",
       "      <td>294</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>222</th>\n",
       "      <td>VARIANT_IS_CLINICALLY_RELEVANT</td>\n",
       "      <td>OncoKB</td>\n",
       "      <td>44</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "                            edge_type  \\\n",
       "0                             ACTS_ON   \n",
       "1                ANNOTATED_IN_PATHWAY   \n",
       "2                ANNOTATED_IN_PATHWAY   \n",
       "3                     ASSOCIATED_WITH   \n",
       "4                     ASSOCIATED_WITH   \n",
       "5                     ASSOCIATED_WITH   \n",
       "6                     ASSOCIATED_WITH   \n",
       "7                     ASSOCIATED_WITH   \n",
       "8                     ASSOCIATED_WITH   \n",
       "9                     ASSOCIATED_WITH   \n",
       "10                    ASSOCIATED_WITH   \n",
       "11                    ASSOCIATED_WITH   \n",
       "12                    ASSOCIATED_WITH   \n",
       "13                    ASSOCIATED_WITH   \n",
       "14                    ASSOCIATED_WITH   \n",
       "15                    ASSOCIATED_WITH   \n",
       "16                    ASSOCIATED_WITH   \n",
       "17                    ASSOCIATED_WITH   \n",
       "18                    ASSOCIATED_WITH   \n",
       "19                    ASSOCIATED_WITH   \n",
       "20                    ASSOCIATED_WITH   \n",
       "21                    ASSOCIATED_WITH   \n",
       "22                    ASSOCIATED_WITH   \n",
       "23                    ASSOCIATED_WITH   \n",
       "24                    ASSOCIATED_WITH   \n",
       "25                    ASSOCIATED_WITH   \n",
       "26                    ASSOCIATED_WITH   \n",
       "27                    ASSOCIATED_WITH   \n",
       "28                    ASSOCIATED_WITH   \n",
       "29                    ASSOCIATED_WITH   \n",
       "30                    ASSOCIATED_WITH   \n",
       "31                    ASSOCIATED_WITH   \n",
       "32                    ASSOCIATED_WITH   \n",
       "33                    ASSOCIATED_WITH   \n",
       "34                    ASSOCIATED_WITH   \n",
       "35                    ASSOCIATED_WITH   \n",
       "36                    ASSOCIATED_WITH   \n",
       "37                    ASSOCIATED_WITH   \n",
       "38                    ASSOCIATED_WITH   \n",
       "39                    ASSOCIATED_WITH   \n",
       "40                    ASSOCIATED_WITH   \n",
       "41                    ASSOCIATED_WITH   \n",
       "42                    ASSOCIATED_WITH   \n",
       "43                    ASSOCIATED_WITH   \n",
       "44                    ASSOCIATED_WITH   \n",
       "45                    ASSOCIATED_WITH   \n",
       "46                    ASSOCIATED_WITH   \n",
       "47                    ASSOCIATED_WITH   \n",
       "48                    ASSOCIATED_WITH   \n",
       "49                    ASSOCIATED_WITH   \n",
       "50                    ASSOCIATED_WITH   \n",
       "51                    ASSOCIATED_WITH   \n",
       "52                    ASSOCIATED_WITH   \n",
       "53                    ASSOCIATED_WITH   \n",
       "54                    ASSOCIATED_WITH   \n",
       "55                    ASSOCIATED_WITH   \n",
       "56                    ASSOCIATED_WITH   \n",
       "57                    ASSOCIATED_WITH   \n",
       "58                    ASSOCIATED_WITH   \n",
       "59                    ASSOCIATED_WITH   \n",
       "60                    ASSOCIATED_WITH   \n",
       "61                    ASSOCIATED_WITH   \n",
       "62                    ASSOCIATED_WITH   \n",
       "63                 BELONGS_TO_PROTEIN   \n",
       "64                 BELONGS_TO_SUBJECT   \n",
       "65            COMPILED_INTERACTS_WITH   \n",
       "66   CURATED_AFFECTS_INTERACTION_WITH   \n",
       "67             CURATED_INTERACTS_WITH   \n",
       "68             CURATED_INTERACTS_WITH   \n",
       "69             CURATED_INTERACTS_WITH   \n",
       "70             CURATED_INTERACTS_WITH   \n",
       "71             CURATED_INTERACTS_WITH   \n",
       "72             CURATED_INTERACTS_WITH   \n",
       "73             CURATED_INTERACTS_WITH   \n",
       "74             CURATED_INTERACTS_WITH   \n",
       "75             CURATED_INTERACTS_WITH   \n",
       "76             CURATED_INTERACTS_WITH   \n",
       "77             CURATED_INTERACTS_WITH   \n",
       "78             CURATED_INTERACTS_WITH   \n",
       "79             CURATED_INTERACTS_WITH   \n",
       "80             CURATED_INTERACTS_WITH   \n",
       "81             CURATED_INTERACTS_WITH   \n",
       "82             CURATED_INTERACTS_WITH   \n",
       "83             CURATED_INTERACTS_WITH   \n",
       "84             CURATED_INTERACTS_WITH   \n",
       "85             CURATED_INTERACTS_WITH   \n",
       "86             CURATED_INTERACTS_WITH   \n",
       "87             CURATED_INTERACTS_WITH   \n",
       "88             CURATED_INTERACTS_WITH   \n",
       "89             CURATED_INTERACTS_WITH   \n",
       "90             CURATED_INTERACTS_WITH   \n",
       "91             CURATED_INTERACTS_WITH   \n",
       "92             CURATED_INTERACTS_WITH   \n",
       "93             CURATED_INTERACTS_WITH   \n",
       "94             CURATED_INTERACTS_WITH   \n",
       "95             CURATED_INTERACTS_WITH   \n",
       "96             CURATED_INTERACTS_WITH   \n",
       "97             CURATED_INTERACTS_WITH   \n",
       "98             CURATED_INTERACTS_WITH   \n",
       "99             CURATED_INTERACTS_WITH   \n",
       "100            CURATED_INTERACTS_WITH   \n",
       "101            CURATED_INTERACTS_WITH   \n",
       "102            CURATED_INTERACTS_WITH   \n",
       "103            CURATED_INTERACTS_WITH   \n",
       "104            CURATED_INTERACTS_WITH   \n",
       "105            CURATED_INTERACTS_WITH   \n",
       "106            CURATED_INTERACTS_WITH   \n",
       "107            CURATED_INTERACTS_WITH   \n",
       "108            CURATED_INTERACTS_WITH   \n",
       "109            CURATED_INTERACTS_WITH   \n",
       "110            CURATED_INTERACTS_WITH   \n",
       "111            CURATED_INTERACTS_WITH   \n",
       "112            CURATED_INTERACTS_WITH   \n",
       "113            CURATED_INTERACTS_WITH   \n",
       "114            CURATED_INTERACTS_WITH   \n",
       "115            CURATED_INTERACTS_WITH   \n",
       "116            CURATED_INTERACTS_WITH   \n",
       "117            CURATED_INTERACTS_WITH   \n",
       "118            CURATED_INTERACTS_WITH   \n",
       "119            CURATED_INTERACTS_WITH   \n",
       "120            CURATED_INTERACTS_WITH   \n",
       "121            CURATED_INTERACTS_WITH   \n",
       "122            CURATED_INTERACTS_WITH   \n",
       "123            CURATED_INTERACTS_WITH   \n",
       "124            CURATED_INTERACTS_WITH   \n",
       "125            CURATED_INTERACTS_WITH   \n",
       "126            CURATED_INTERACTS_WITH   \n",
       "127            CURATED_INTERACTS_WITH   \n",
       "128            CURATED_INTERACTS_WITH   \n",
       "129            CURATED_INTERACTS_WITH   \n",
       "130            CURATED_INTERACTS_WITH   \n",
       "131            CURATED_INTERACTS_WITH   \n",
       "132            CURATED_INTERACTS_WITH   \n",
       "133            CURATED_INTERACTS_WITH   \n",
       "134            CURATED_INTERACTS_WITH   \n",
       "135            CURATED_INTERACTS_WITH   \n",
       "136            CURATED_INTERACTS_WITH   \n",
       "137            CURATED_INTERACTS_WITH   \n",
       "138            CURATED_INTERACTS_WITH   \n",
       "139            CURATED_INTERACTS_WITH   \n",
       "140            CURATED_INTERACTS_WITH   \n",
       "141            CURATED_INTERACTS_WITH   \n",
       "142            CURATED_INTERACTS_WITH   \n",
       "143            CURATED_INTERACTS_WITH   \n",
       "144            CURATED_INTERACTS_WITH   \n",
       "145            CURATED_INTERACTS_WITH   \n",
       "146            CURATED_INTERACTS_WITH   \n",
       "147            CURATED_INTERACTS_WITH   \n",
       "148            CURATED_INTERACTS_WITH   \n",
       "149            CURATED_INTERACTS_WITH   \n",
       "150            CURATED_INTERACTS_WITH   \n",
       "151            CURATED_INTERACTS_WITH   \n",
       "152            CURATED_INTERACTS_WITH   \n",
       "153            CURATED_INTERACTS_WITH   \n",
       "154            CURATED_INTERACTS_WITH   \n",
       "155            CURATED_INTERACTS_WITH   \n",
       "156            CURATED_INTERACTS_WITH   \n",
       "157            CURATED_INTERACTS_WITH   \n",
       "158            CURATED_INTERACTS_WITH   \n",
       "159            CURATED_INTERACTS_WITH   \n",
       "160            CURATED_INTERACTS_WITH   \n",
       "161            CURATED_INTERACTS_WITH   \n",
       "162            CURATED_INTERACTS_WITH   \n",
       "163            CURATED_INTERACTS_WITH   \n",
       "164            CURATED_INTERACTS_WITH   \n",
       "165            CURATED_INTERACTS_WITH   \n",
       "166            CURATED_INTERACTS_WITH   \n",
       "167            CURATED_INTERACTS_WITH   \n",
       "168            CURATED_INTERACTS_WITH   \n",
       "169            CURATED_INTERACTS_WITH   \n",
       "170            CURATED_INTERACTS_WITH   \n",
       "171            CURATED_INTERACTS_WITH   \n",
       "172            CURATED_INTERACTS_WITH   \n",
       "173            CURATED_INTERACTS_WITH   \n",
       "174            CURATED_INTERACTS_WITH   \n",
       "175            CURATED_INTERACTS_WITH   \n",
       "176            CURATED_INTERACTS_WITH   \n",
       "177            CURATED_INTERACTS_WITH   \n",
       "178            CURATED_INTERACTS_WITH   \n",
       "179            CURATED_INTERACTS_WITH   \n",
       "180            CURATED_INTERACTS_WITH   \n",
       "181            CURATED_INTERACTS_WITH   \n",
       "182            CURATED_INTERACTS_WITH   \n",
       "183            CURATED_INTERACTS_WITH   \n",
       "184            CURATED_INTERACTS_WITH   \n",
       "185            CURATED_INTERACTS_WITH   \n",
       "186            CURATED_INTERACTS_WITH   \n",
       "187            CURATED_INTERACTS_WITH   \n",
       "188            CURATED_INTERACTS_WITH   \n",
       "189      DETECTED_IN_PATHOLOGY_SAMPLE   \n",
       "190                  FOUND_IN_PROTEIN   \n",
       "191                      HAS_ENROLLED   \n",
       "192                  HAS_MODIFICATION   \n",
       "193                  HAS_MODIFICATION   \n",
       "194                 HAS_MODIFIED_SITE   \n",
       "195                        HAS_PARENT   \n",
       "196   HAS_QUANTIFIED_MODIFIED_PROTEIN   \n",
       "197            HAS_QUANTIFIED_PROTEIN   \n",
       "198                      HAS_SEQUENCE   \n",
       "199                     HAS_STRUCTURE   \n",
       "200           IS_BIOMARKER_OF_DISEASE   \n",
       "201             IS_QCMARKER_IN_TISSUE   \n",
       "202                    IS_RESPONSIBLE   \n",
       "203                   IS_SUBSTRATE_OF   \n",
       "204                     IS_SUBUNIT_OF   \n",
       "205                        LOCATED_IN   \n",
       "206                           MAPS_TO   \n",
       "207          MENTIONED_IN_PUBLICATION   \n",
       "208          MENTIONED_IN_PUBLICATION   \n",
       "209                   PARTICIPATES_IN   \n",
       "210                      PUBLISHED_IN   \n",
       "211                     SPLITTED_INTO   \n",
       "212                   STUDIES_DISEASE   \n",
       "213                    STUDIES_TISSUE   \n",
       "214                     STUDIES_TRAIT   \n",
       "215                  TRANSCRIBED_INTO   \n",
       "216                   TRANSLATED_INTO   \n",
       "217       VARIANT_FOUND_IN_CHROMOSOME   \n",
       "218             VARIANT_FOUND_IN_GENE   \n",
       "219             VARIANT_FOUND_IN_GWAS   \n",
       "220          VARIANT_FOUND_IN_PROTEIN   \n",
       "221    VARIANT_IS_CLINICALLY_RELEVANT   \n",
       "222    VARIANT_IS_CLINICALLY_RELEVANT   \n",
       "\n",
       "                                                source      count  \n",
       "0                                               STRING    1977410  \n",
       "1                                             Reactome     117270  \n",
       "2                                                SMPDB    2290348  \n",
       "3                           DisGeNet: ORPHANET;UNIPROT         52  \n",
       "4    DisGeNet: CLINGEN;GENOMICS_ENGLAND;ORPHANET;UN...         12  \n",
       "5                                DisGeNet: CGI;UNIPROT        106  \n",
       "6                       DisGeNet: CGI;GENOMICS_ENGLAND        128  \n",
       "7     DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLAND         10  \n",
       "8           DisGeNet: CLINGEN;GENOMICS_ENGLAND;UNIPROT         22  \n",
       "9               DisGeNet: CGI;GENOMICS_ENGLAND;UNIPROT         10  \n",
       "10                              DisGeNet: CGI;ORPHANET         34  \n",
       "11                                  DisGeNet: PSYGENET       9634  \n",
       "12                 DisGeNet: CLINGEN;CTD_human;UNIPROT          4  \n",
       "13   DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;U...        226  \n",
       "14   DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;O...        298  \n",
       "15                    DisGeNet: CGI;CTD_human;ORPHANET         66  \n",
       "16        DisGeNet: CLINGEN;CTD_human;ORPHANET;UNIPROT          2  \n",
       "17   DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;ORPHA...          8  \n",
       "18         DisGeNet: GENOMICS_ENGLAND;ORPHANET;UNIPROT        178  \n",
       "19   DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;ORPHANET          8  \n",
       "20                     DisGeNet: CGI;CLINGEN;CTD_human          4  \n",
       "21            DisGeNet: CGI;CLINGEN;CTD_human;ORPHANET          6  \n",
       "22                           DisGeNet: CLINGEN;UNIPROT          2  \n",
       "23   DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLA...          4  \n",
       "24                 DisGeNet: GENOMICS_ENGLAND;PSYGENET          2  \n",
       "25            DisGeNet: CGI;CTD_human;ORPHANET;UNIPROT          2  \n",
       "26   DisGeNet: CGI;CLINGEN;CTD_human;GENOMICS_ENGLA...          4  \n",
       "27                                             TISSUES   13058452  \n",
       "28                                             UniProt     672516  \n",
       "29                                               CORUM       9540  \n",
       "30                                        COMPARTMENTS    7124610  \n",
       "31                                              OncoKB        186  \n",
       "32                         DisGeNet: CLINGEN;CTD_human        296  \n",
       "33                        DisGeNet: CTD_human;PSYGENET        566  \n",
       "34                DisGeNet: CLINGEN;CTD_human;ORPHANET         90  \n",
       "35                                            DISEASES    8931224  \n",
       "36                                                 CGI       8924  \n",
       "37                                    DisGeNet: BEFREE    1688338  \n",
       "38                                 DisGeNet: CTD_human      87860  \n",
       "39                             DisGeNet: CGI;CTD_human        802  \n",
       "40                          DisGeNet: GENOMICS_ENGLAND      14804  \n",
       "41                                  DisGeNet: ORPHANET       7964  \n",
       "42                        DisGeNet: CTD_human;ORPHANET       1590  \n",
       "43        DisGeNet: CTD_human;GENOMICS_ENGLAND;UNIPROT       2598  \n",
       "44         DisGeNet: CLINGEN;GENOMICS_ENGLAND;ORPHANET        248  \n",
       "45                 DisGeNet: GENOMICS_ENGLAND;ORPHANET       1306  \n",
       "46   DisGeNet: CTD_human;GENOMICS_ENGLAND;ORPHANET;...       2530  \n",
       "47                DisGeNet: CTD_human;GENOMICS_ENGLAND       3090  \n",
       "48                                   DisGeNet: CLINGEN       3392  \n",
       "49                                   DisGeNet: UNIPROT       5028  \n",
       "50       DisGeNet: CTD_human;GENOMICS_ENGLAND;ORPHANET       1548  \n",
       "51                          DisGeNet: CLINGEN;ORPHANET        358  \n",
       "52                  DisGeNet: CLINGEN;GENOMICS_ENGLAND        186  \n",
       "53                                       DisGeNet: CGI       2506  \n",
       "54            DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND         54  \n",
       "55                         DisGeNet: CTD_human;UNIPROT        662  \n",
       "56                     DisGeNet: CGI;CTD_human;UNIPROT        126  \n",
       "57   DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND;O...        260  \n",
       "58        DisGeNet: CLINGEN;CTD_human;GENOMICS_ENGLAND        216  \n",
       "59                                                None    1771568  \n",
       "60                DisGeNet: CTD_human;ORPHANET;UNIPROT        152  \n",
       "61                  DisGeNet: GENOMICS_ENGLAND;UNIPROT        784  \n",
       "62    DisGeNet: CGI;CTD_human;GENOMICS_ENGLAND;UNIPROT         62  \n",
       "63                                             UniProt    7258116  \n",
       "64                                                None        340  \n",
       "65                                            [STRING]    3913224  \n",
       "66                                   Intact-MutationDs      21746  \n",
       "67                       [IntAct, DIP, InnateDB, MINT]          4  \n",
       "68                                [UniProt, DIP, MINT]         37  \n",
       "69                                  [IntAct, matrixdb]          6  \n",
       "70                    [UniProt, IntAct, matrixdb, DIP]          4  \n",
       "71          [Molecular Connections, IntAct, I2D, MINT]          2  \n",
       "72                               [DIP, InnateDB, MINT]          8  \n",
       "73                                   [UniProt, mbinfo]          2  \n",
       "74                                [bhf-ucl, DIP, MINT]          3  \n",
       "75     [I2D, IntAct, Molecular Connections, DIP, MINT]          2  \n",
       "76                       [IntAct, matrixdb, DIP, MINT]          2  \n",
       "77                                  [IntAct, InnateDB]         18  \n",
       "78                        [Molecular Connections, DIP]          6  \n",
       "79                                       [IntAct, I2D]         30  \n",
       "80                        [bhf-ucl, I2D, IntAct, MINT]          2  \n",
       "81                        [bhf-ucl, IntAct, DIP, MINT]          8  \n",
       "82                                     [IntAct, HPIDb]         82  \n",
       "83                                    [UniProt, HPIDb]         74  \n",
       "84                        [bhf-ucl, IntAct, I2D, MINT]          6  \n",
       "85                      [HPIDb, UniProt, IntAct, MINT]          6  \n",
       "86                               [HPIDb, IntAct, MINT]         16  \n",
       "87                            [UniProt, IntAct, HPIDb]         25  \n",
       "88                               [UniProt, DIP, HPIDb]          2  \n",
       "89                              [HPIDb, UniProt, MINT]         10  \n",
       "90          [Molecular Connections, IntAct, DIP, MINT]         10  \n",
       "91                     [Molecular Connections, IntAct]         58  \n",
       "92                                  [I2D, IntAct, DIP]          6  \n",
       "93                                       [HPIDb, MINT]         18  \n",
       "94                                        [DIP, HPIDb]          5  \n",
       "95                            [IntAct, I2D, DIP, MINT]         16  \n",
       "96                   [IntAct, I2D, DIP, MINT, bhf-ucl]          4  \n",
       "97        [Molecular Connections, IntAct, MINT, HPIDb]          2  \n",
       "98    [UniProt, Molecular Connections, IntAct, mbinfo]          2  \n",
       "99                [Molecular Connections, IntAct, DIP]          5  \n",
       "100             [Molecular Connections, IntAct, HPIDb]          6  \n",
       "101                                            [mpidb]          6  \n",
       "102                               [IntAct, DIP, HPIDb]          7  \n",
       "103                            [IntAct, matrixdb, DIP]          5  \n",
       "104                   [Molecular Connections, bhf-ucl]          2  \n",
       "105                                   [bhf-ucl, HPIDb]          1  \n",
       "106                          [UniProt, IntAct, mbinfo]          2  \n",
       "107                                    [DIP, matrixdb]          2  \n",
       "108                                   [I2D, DIP, MINT]          2  \n",
       "109                           [I2D, IntAct, DIP, MINT]          2  \n",
       "110               [Molecular Connections, I2D, IntAct]          2  \n",
       "111          [Molecular Connections, IntAct, DIP, I2D]          8  \n",
       "112                                [IntAct, I2D, MINT]         24  \n",
       "113     [Molecular Connections, bhf-ucl, IntAct, MINT]          2  \n",
       "114                       [UniProt, IntAct, I2D, MINT]          2  \n",
       "115                        [UniProt, IntAct, InnateDB]          4  \n",
       "116                           [IntAct, InnateDB, MINT]          6  \n",
       "117              [bhf-ucl, Molecular Connections, I2D]          2  \n",
       "118                 [NTNU, IntAct, I2D, MINT, bhf-ucl]          2  \n",
       "119                                [bhf-ucl, I2D, DIP]          2  \n",
       "120              [IntAct, UniProt, DIP, MINT, bhf-ucl]          1  \n",
       "121                                 [IntAct, DIP, I2D]          5  \n",
       "122           [bhf-ucl, Molecular Connections, IntAct]          2  \n",
       "123                                    [matrixdb, DIP]          2  \n",
       "124                                         [I2D, DIP]         12  \n",
       "125                      [UniProt, IntAct, DIP, HPIDb]          4  \n",
       "126                             [mbinfo, IntAct, MINT]          6  \n",
       "127                                   [UniProt, mpidb]          1  \n",
       "128                                [UniProt, InnateDB]          2  \n",
       "129                                         [matrixdb]        293  \n",
       "130                                      [IntAct, DIP]       1534  \n",
       "131                         [HPIDb, IntAct, DIP, MINT]          6  \n",
       "132                                     [mbinfo, MINT]         12  \n",
       "133      [UniProt, Molecular Connections, IntAct, DIP]          2  \n",
       "134                                          [bhf-ucl]       3332  \n",
       "135                  [UniProt, IntAct, InnateDB, MINT]          2  \n",
       "136                                     [IntAct, MINT]       5360  \n",
       "137                 [Molecular Connections, DIP, MINT]          2  \n",
       "138                                              [DIP]       9419  \n",
       "139                                              [I2D]        734  \n",
       "140                                             [MINT]      55665  \n",
       "141             [IntAct, UniProt, DIP, InnateDB, MINT]          3  \n",
       "142                                          [UniProt]      19157  \n",
       "143                                           [IntAct]     494635  \n",
       "144                                      [mbinfo, DIP]          2  \n",
       "145                           [IntAct, MINT, DIP, I2D]          2  \n",
       "146                     [Molecular Connections, HPIDb]          6  \n",
       "147                                      [MINT, HPIDb]          4  \n",
       "148                  [IntAct, I2D, UniProt, DIP, MINT]          2  \n",
       "149                                   [InnateDB, MINT]          2  \n",
       "150                            [IntAct, InnateDB, I2D]          2  \n",
       "151                [MINT, IntAct, UniProt, DIP, HPIDb]          3  \n",
       "152           [MINT, IntAct, UniProt, InnateDB, HPIDb]          2  \n",
       "153                                   [matrixdb, MINT]          3  \n",
       "154                   [UniProt, Molecular Connections]          3  \n",
       "155                              [mbinfo, IntAct, DIP]          4  \n",
       "156                             [UniProt, HPIDb, MINT]          1  \n",
       "157                                  [bhf-ucl, IntAct]         86  \n",
       "158                             [bhf-ucl, IntAct, DIP]         16  \n",
       "159                                [IntAct, DIP, MINT]        446  \n",
       "160                                        [DIP, MINT]        361  \n",
       "161                                             [NTNU]         46  \n",
       "162                                     [bhf-ucl, DIP]         20  \n",
       "163                                 [UniProt, bhf-ucl]          9  \n",
       "164                            [bhf-ucl, IntAct, MINT]         26  \n",
       "165                                [I2D, IntAct, MINT]          8  \n",
       "166                            [IntAct, DIP, InnateDB]          8  \n",
       "167              [Molecular Connections, IntAct, MINT]         17  \n",
       "168                   [UniProt, IntAct, bhf-ucl, MINT]          2  \n",
       "169                       [UniProt, IntAct, DIP, MINT]        133  \n",
       "170                               [NTNU, IntAct, MINT]          4  \n",
       "171                                      [I2D, IntAct]         40  \n",
       "172                                  [UniProt, IntAct]       1413  \n",
       "173                                         [InnateDB]        291  \n",
       "174                                     [UniProt, DIP]        151  \n",
       "175                       [Molecular Connections, I2D]          2  \n",
       "176                                        [I2D, MINT]         38  \n",
       "177                             [UniProt, IntAct, DIP]        170  \n",
       "178                                    [UniProt, MINT]        473  \n",
       "179                                   [mbinfo, IntAct]         17  \n",
       "180                                    [bhf-ucl, MINT]         32  \n",
       "181                                           [mbinfo]        209  \n",
       "182                    [mbinfo, IntAct, bhf-ucl, MINT]          2  \n",
       "183                                    [DIP, InnateDB]          7  \n",
       "184                            [Molecular Connections]        336  \n",
       "185                        [UniProt, IntAct, DIP, I2D]          2  \n",
       "186                      [Molecular Connections, MINT]         22  \n",
       "187                            [UniProt, IntAct, MINT]        350  \n",
       "188                                            [HPIDb]        427  \n",
       "189                      Human Protein Atlas pathology    3394496  \n",
       "190                                               PFam     408488  \n",
       "191                                               None        338  \n",
       "192                                               None      41572  \n",
       "193                                             SIGNOR       1242  \n",
       "194                                               None      42842  \n",
       "195                                               None     256698  \n",
       "196                                               None     448956  \n",
       "197                                               None    1595302  \n",
       "198                                            UniProt      41228  \n",
       "199                                            UniProt     391280  \n",
       "200                    Nicolai Jacob Wewer Albrechtsen       1030  \n",
       "201                                               None        498  \n",
       "202                                               None         14  \n",
       "203                                             SIGNOR       1988  \n",
       "204                                              CORUM      21936  \n",
       "205                                               None     591824  \n",
       "206                                               None       4578  \n",
       "207                                               PFam       4870  \n",
       "208                                               None  222213606  \n",
       "209                                               None         14  \n",
       "210                                               None       7878  \n",
       "211                                               None        344  \n",
       "212                                               None         14  \n",
       "213                                               None         14  \n",
       "214                                       GWAS Catalog      18500  \n",
       "215                                               None     516974  \n",
       "216                                               None     748588  \n",
       "217                                               None   21260216  \n",
       "218                                               None   21277870  \n",
       "219                                       GWAS Catalog      32256  \n",
       "220                                               None   53614586  \n",
       "221                                                CGI        294  \n",
       "222                                             OncoKB         44  "
      ]
     },
     "execution_count": 12,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "pd.set_option('display.max_rows', None)\n",
    "edge_sources_table"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "036972a3-c347-418c-be31-04ac2ef7dac7",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.5"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
